client/src/components/Modals/EditInterests/index.tsx

```diff
@@ -0,0 +1,200 @@
+import React, { ReactElement, FC, useState, useEffect } from 'react';
```

Comment:
This is also a copy paste of the teams stuff, can we combine them into 1 generic one and then use each separately as components?